src/qz/common/TrayManager.java

```diff
@@ -156,101 +156,94 @@ public static void main(String args[]) {
+        tray.getMenu().add(advancedMenu);
+        {
+            MenuItem sitesItem = new MenuItem("Site Manager...", iconCache.getImage(IconCache.Icon.SAVED_ICON), savedListener);
+            sitesItem.setShortcut('m');
+            advancedMenu.add(sitesItem);
+
+            Checkbox anonymousItem = new Checkbox("Block Anonymous Requests", anonymousListener);
+            anonymousItem.setShortcut('k');
+            anonymousItem.setChecked(Certificate.UNKNOWN.isBlocked());
+            //anonymousItem.setToolTipText("Blocks all requests that do no contain a valid certificate/signature");
+            advancedMenu.add(anonymousItem);
+
+            MenuItem logItem = new MenuItem("View Logs...", iconCache.getImage(IconCache.Icon.LOG_ICON), logListener);
+            sitesItem.setShortcut('l');
+            advancedMenu.add(logItem);
+
+            Checkbox notificationsItem = new Checkbox("Show all notifications", notificationsListener);
+            notificationsItem.setShortcut('s');
+            notificationsItem.setChecked(prefs.getBoolean(Constants.PREFS_NOTIFICATIONS, false));
+            //anonymousItem.setToolTipText("Blocks all requests that do no contain a valid certificate/signature");
+            advancedMenu.add(notificationsItem);
+
+            MenuItem openItem = new MenuItem("Open file location", iconCache.getImage(IconCache.Icon.FOLDER_ICON), openListener);
+            sitesItem.setShortcut('o');
+            advancedMenu.add(openItem);
+
+            MenuItem desktopItem = new MenuItem("Create Desktop shortcut", iconCache.getImage(IconCache.Icon.DESKTOP_ICON), desktopListener);
+            sitesItem.setShortcut('d');
+            advancedMenu.add(desktopItem);
         }
 
-        JSeparator separator = new JSeparator();
-
-        JCheckBoxMenuItem startupItem = new JCheckBoxMenuItem("Automatically start");
-        startupItem.setMnemonic(KeyEvent.VK_S);
-        startupItem.setState(shortcutCreator.hasStartupShortcut());
-        startupItem.addActionListener(startupListener);
-
-        JMenuItem exitItem = new JMenuItem("Exit", iconCache.getIcon(IconCache.Icon.EXIT_ICON));
-        exitItem.addActionListener(exitListener);
+        MenuItem reloadItem = new MenuItem("Reload", iconCache.getImage(IconCache.Icon.RELOAD_ICON), reloadListener);
+        reloadItem.setShortcut('r');
+        tray.getMenu().add(reloadItem);
+
+        MenuItem aboutItem = new MenuItem("About...", iconCache.getImage(IconCache.Icon.ABOUT_ICON), aboutListener);
+        aboutItem.setShortcut('b');
+        tray.getMenu().add(aboutItem);
+
+        aboutDialog = new AboutDialog(null, iconCache, name);
+        {
+            JMenuItem siteButton = new JMenuItem("Site Manager...", iconCache.getIcon(IconCache.Icon.SAVED_ICON));
+            siteButton.setMnemonic(KeyEvent.VK_M);
+            siteButton.addActionListener(savedListener);
+            sitesDialog = new SiteManagerDialog(siteButton, iconCache);
```

Comment:
Kind of ugly, but if we need buttons on windows, there is no way around it. This code could be moved into aboutDialog but bleh.